### ISSUE TITLE
output strands after root correction artifacts

### ISSUE THREAD

**Issue by kruzel** [pielet/HairNet#10]
Hi
I use this code to output the strands from HairNet output.

    net = Net()
    net.load_state_dict(torch.load(weight_path))
    net.eval()
    tmp_img = GetImgTensorFromInd(proj_dir, files_index, file_ind)
    output_strands = net(tmp_img)
    strand_roots = np.load('strand_root_positions.npy')
    output_strands = output_strands.cpu().detach().numpy()
    output_strands = output_strands[0]
    strand_roots_2 = np.concatenate((strand_roots,np.zeros((1,32,32))))
    strand_roots_3 = strand_roots_2[np.newaxis,:,:,:]
    output_strands = output_strands + strand_roots_3 #brodcast (1,4,32,32) t0 (100,4,32,32)
    WriteStrandsToObjText("../my_hair.obj", output_strands)

but I get 8401 more vertices then the original strands.
you can see the redundant strands in the lower part of this image.
any idea?

![image](https://github.com/pielet/HairNet/assets/975963/23c0afb6-6fd6-4786-9c08-7b0825460d57)

**Comment by kruzel** [comment]
found the problem,
strand root file have only 940 points instead of 1024 (32x32).
how can I fix the strand roots file?
